lib/publiccloud/azure.pm

```diff
@@ -387,7 +387,7 @@ sub cleanup {
 
     my $id = $args->{my_instance}->{instance_id};
 
-    script_run("az vm boot-diagnostics get-boot-log --ids $id | jq -r '.' > bootlog.txt");
+    script_run("az vm boot-diagnostics get-boot-log --ids $id | jq -r '.' > bootlog.txt", timeout => 120, die_on_timeout => 0);
```

Comment:
```suggestion
    script_run("az vm boot-diagnostics get-boot-log --ids $id | jq -r '.' | tee bootlog.txt", timeout => 120, die_on_timeout => 0);
```

Reply:
I'd rather not. The log can be large, and spamming openQA with this doesn't seem to give us any obvious benefit.

Reply:
My point was that the log can contain error / debug messages. But if we proceed and upload the file even in case of failure then it's probably good.